### ISSUE TITLE
cmd/go: TestLldbPython newly failing (related to enabling modules)

### ISSUE THREAD

**Issue by josharian** [golang/go#30751]
On darwin/amd64, running all.bash at tip, I get:

```go
--- FAIL: TestLldbPython (2.12s)
    runtime-lldb_test.go:163: building source exit status 1
        go: cannot find main module; see 'go help modules'
FAIL
FAIL	runtime	26.998s
```

The test passed this morning. :)

cc @bcmills @jayconrod

**Comment by mikioh** [comment]
It might be better to run "/usr/sbin/DevToolsSecurity -enable" on darwin buildbots by default.

**Comment by josharian** [comment]
Sure. But this was on my local machine. And we don’t want to ask everyone who runs all.bash (often only once) to jump through any hoops. So the test still needs to be resilient.

**Comment by bradfitz** [comment]
What does DevToolsSecurity -enable have to do with this?

**Comment by bcmills** [comment]
> On darwin/amd64, running all.bash at tip

Do we know why this issue didn't show up in the builders? (Should we expand the darwin/amd64 builders in some way to enable them to catch regressions of this test?)

**Comment by josharian** [comment]
> Do we know why this issue didn't show up in the builders? (Should we expand the darwin/amd64 builders in some way to enable them to catch regressions of this test?)

Good question. Maybe that's why @mikioh meant by suggesting that we run `/usr/sbin/DevToolsSecurity -enable` on the builders. I'll open a new issue for that a bit later (have to run now).
